src/diffusers/pipeline_utils.py

```diff
@@ -71,6 +71,7 @@
         "SchedulerMixin": ["save_pretrained", "from_pretrained"],
         "DiffusionPipeline": ["save_pretrained", "from_pretrained"],
         "OnnxRuntimeModel": ["save_pretrained", "from_pretrained"],
+        "Mel": ["save_pretrained", "from_pretrained"],
```

Comment:
A bit too specific to go into here, let's better move it to the pipelines :-)

Reply:
Done.